### DISCUSSION TITLE
Can I safely set `z-index: auto` to `[data-overlayscrollbars-viewport]` element?

### DISCUSSION THREAD

**Discussion by hatttfu** [KingSora/OverlayScrollbars#649] [Q&A] [answered]
Firstly, thank you for this beautiful library!

**Describe the bug**
I have some fixed block with z-index: 10 and with scrollable content inside. Also I've got one more fixed element inside this scrollable content, which overlapping by scrollbar. It can be solved by setting `z-index: auto` to [data-overlayscrollbars-viewport] element, but I'm not sure it doesn't have any unintended consenquences.

**Examples**
https://codesandbox.io/p/devbox/overlayscrollbars-vue-example-forked-d4jljx?file=%2Fsrc%2FApp.css%3A11%2C19

**To Reproduce**

1. Go to block and scroll down  
2. See block being overlapped by scrollbar

**Comment by KingSora** [comment, marked as answer]
Good day @hatttfu :)

Changing `z-index` should be purely cosmetic change and if it works for your usecase there shouldn't speak anything against it

**Reply by hatttfu** [reply]
Thank you!
